cadc-registry/build.gradle

```diff
@@ -15,7 +15,7 @@ sourceCompatibility = 11
 
 group = 'org.opencadc'
 
-version = '1.8.0'
+version = '1.8.1'
```

Comment:
I think you could leave this unchanged. I'll probably merge both PRs before publishing